packages/db/test/extensions/stats.test.ts

```diff
@@ -0,0 +1,267 @@
+import { describe, it, expect } from "vitest";
+
+import { prisma } from "../../prisma";
+
+describe("Stats Extension", () => {
+  describe("Daily Stats", () => {
+    describe("Blob model", () => {
+      it("should fill stats in a date period", async () => {
```

Comment:
Right now it will fail if we do it, but we should check for different kind of date input formats and expect the operation to run.